Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
> I guess you use multiple @covers annotations in that case.

Yes if you don't forgot some. This is why I'm not a big fan human error with that are too risky IMHO.

This is my vote, but not a veto of course.